release_docs/RELEASE.txt

```diff
@@ -301,6 +301,14 @@ Bug Fixes since HDF5-1.13.3 release
 
     Configuration
     -------------
+    - Change the default for building the high-level tools
```

Comment:
I wonder if it might be good to call out which tools are getting disabled here by default since I imagine some folks would want to know that building of h5watch is opt-in after this.

Reply:
Agree. The tools are gif2hdf and hdf2gif as well as h5watch

Reply:
done